TMinus/Launch.swift

```diff
@@ -29,6 +30,31 @@ struct Launch {
     //MARK: Private
 }
 
+extension Launch: NotificationType {
+    
+    func makeNotificationRequest() -> UNNotificationRequest {
+        let date = startDate.addingTimeInterval(-.oneHour)
+        let components = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second], from: date)
+        
+        let content = UNMutableNotificationContent()
+        var title = rocketName
+        if let missionName = missionName, !missionName.isEmpty {
+            title = "\(title) - \(missionName)"
+        }
+        content.title = title
+        content.body = "The launch window is scheduled to open in one hour."
```

Comment:
I'm cool to localize. I don't have a ton of experience with it, though. We can talk about the best approach.